benches/src/extract.rs

```diff
@@ -6,7 +6,7 @@ use std::path::PathBuf;
 fn extract(paths: &[PathBuf]) {
     for path in paths {
         let content: InputContent = path.try_into().unwrap();
-        let extracted = Extractor::extract(&content);
+        let extracted = Extractor::extract(&content, false);
```

Comment:
Came back for the final review and had a hard time remembering what `false` was for. An alternative would be to have `Extract::with_html5gum` and `Extract::with_html5ever`. More explicit without using a config struct. What do you think?

Reply:
i did something like that now.. it's a bit repetitive in both Extractor and usages of it, but in return we have no breaking api change